src/components/tracked-controls.js

```diff
@@ -249,7 +269,7 @@ module.exports.Component = registerComponent('tracked-controls', {
     for (i = 0; i < controllerAxes.length; i++) {
       this.axis.push(controllerAxes[i]);
     }
-    this.el.emit('axismove', this.axisMoveEventDetail);
+    this.el.emit(EVENTS.AXISMOVE, this.axisMoveEventDetail, false);
```

Comment:
I’m ok with the PR but I believe these strings should be allocated on stack and not on the heap so no subjects for GC

Reply:
I read [strings get collected](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Memory_Management). But this PR also removes the string concatenations which do allocate and are messy anyways, constants are cleaner.